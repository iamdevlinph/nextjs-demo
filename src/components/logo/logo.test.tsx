import { shallow } from 'enzyme';
import React from 'react';

import { Logo } from './index';

describe('Logo Component', () => {
  it('should match logo snapshot if no props provided', () => {
    const wrapper = shallow(<Logo />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match logo snapshot if props.isSquare', () => {
    const wrapper = shallow(<Logo isSquare />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match logo snapshot if props.isSquare={false}', () => {
    const wrapper = shallow(<Logo isSquare={false} />);
    expect(wrapper).toMatchSnapshot();
  });

  it('should match logo snapshot if props.className', () => {
    const wrapper = shallow(<Logo className="block" />);
    expect(wrapper).toMatchSnapshot();
  });
});
