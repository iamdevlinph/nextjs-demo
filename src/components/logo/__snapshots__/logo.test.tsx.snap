// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo Component should match logo snapshot if no props provided 1`] = `
<img
  alt="logo"
  className=""
  src="/images/logo.svg"
/>
`;

exports[`Logo Component should match logo snapshot if props.className 1`] = `
<img
  alt="logo"
  className="block"
  src="/images/logo.svg"
/>
`;

exports[`Logo Component should match logo snapshot if props.isSquare 1`] = `
<img
  alt="logo"
  className=""
  src="/images/logo.svg"
/>
`;

exports[`Logo Component should match logo snapshot if props.isSquare={false} 1`] = `
<img
  alt="logo"
  className=""
  src="/images/logo-no-margin.svg"
/>
`;
